src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.text.DecimalFormat;
+
+public abstract class Figure implements QuadrangularArea, TriangleArea {
+    protected static final String DECIMAL_FORMAT_PATTERN = "#.00";
+    protected DecimalFormat decimalFormat = new DecimalFormat(DECIMAL_FORMAT_PATTERN);
+
+    protected String color;
+    protected TypeOfFigure typeOfFigure;
+    protected double area;
+
+    public abstract void drawFigure();
```

Comment:
The method name `drawFigure` could be simplified to `draw` since it's already in the context of a `Figure` class. Method names should be verbs and should clearly represent the action they perform.